AuthPackage/CustomBearerHandler.cs

```diff
@@ -41,27 +41,28 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
             var signingKeys = await _publicKeyService.GetSigningKeysFromJwkAsync(_fdcOptions.SigningKeysUri);
 
             // Configure token validation parameters
-            var validationParameters = new TokenValidationParameters
-            {
-                ValidateIssuer = true,
-                ValidateAudience = true,
-                ValidateLifetime = true,
-                ValidateIssuerSigningKey = true,
-                IssuerSigningKeys = signingKeys,
 
-                // Set the fetched signing keys
-                ValidIssuer = Options.Authority,
-                ValidAudience = Options.Audience
-            };
+            // Removed below part since we are already doing it in Protected resource and extensions method
```

Comment:
removed this TokenValidationParameters options here as I felt like they are duplicate